notNeededPackages.json

```
@@ -5183,6 +5183,10 @@
             "libraryName": "react-inlinesvg",
             "asOfVersion": "1.0.0"
         },
+        "react-inspector": {
+            "libraryName": "react-inspector",
+            "asOfVersion": "6.0.0"
```

```suggestion
            "asOfVersion": "6.0.1"
```

6.0.0 doesn't seem to have any types in it.

---

Done.